readme.md

```diff
@@ -30,6 +31,58 @@ class Example extends Component {
 }
 ```
 
+*Custom Form using json schema and UI schema*
+```jsx
+import React, { Component } from 'react';
+
+import { CustomForm } from 'dynamic-forms-react';
+
+class Example extends Component {
+  render () {
+    return (
+      <CustomForm
+        formUrl={this.paramUrl('id')}
+        onSubmit={(res) => (alert(res))}
+        onError={(error) => alert(error)}
+      />
+    )
+  }
+}
+```
+
+This is how you need to pass the json schema from your endpoint:
```

Comment:
I think it should be most like " The URL should contain the next structure "

```
{
  json_schema:{
    ...
  },
  ui_schima:{
    ...
  }
}

```
here an example

```
{
  json_schema:{
    "title": "Name Form",
    "description": "A simple name form ",
    "type": "object",
    "required": [
      "name",
    ],
    "additionalProperties": false,
    "properties": {
      "name": {
        "type": "string",
        "title": "Name"
      }
    }
  },
  ui_schima:{
    "name": {
      "ui:autofocus": true,
      "ui:title": "First and Last Name",
      "ui:emptyValue": ""
    },
  }
}

```
because you component receive a URL that it contains a JSON

Reply:
we should use ui_schema